components/Bubbles/BubblesGame.js

```diff
@@ -226,8 +254,9 @@ class BubblesGame extends React.Component {
 
     clearInterval(this.eatInterval)
     this.eatInterval = setInterval(() => {
+      console.warn('eating');
       this.setState({
-        monsterAnimationIndex: [0,4,5,4,0],
+        monsterAnimationIndex: monsterUtils.eatIndx,
```

Comment:
here you should be able to sue the character itself. Something like

monsterRef.animationIndex('EAT')